xwiki-platform-tools/xwiki-platform-tool-configuration-resources/src/main/resources/xwiki.properties.vm

```diff
@@ -1133,4 +1133,8 @@ edit.defaultEditor.org.xwiki.rendering.block.XDOM#wysiwyg=$xwikiPropertiesDefaul
 #-# The default value is:
 # refactoring.rename.useAtomicRename = true
 
+#-# [Since 12.8RC1]
+#-# Define if the reyclebin can be skipped by advanced users when deleting documents.
+# refactoring.recyclebin.skip = false
```

Comment:
I would be more clear on what exactly is the default value.

Reply:
Documentation updated.

Reply:
The name is not that great since it's not about skipping the recycle bin or not. It's about whether it can be skipped. So IMO it should be:

```
refactoring.canSkipRecycleBin = false
```

Reply:
I agree, I'll change that while migrating the config to an XClass.

Reply:
Fully replaced by an XClass. `xwiki.properties.vm` is not modified in this PR anymore.